Gems/ROS2/Code/Include/ROS2/ROS2Bus.h

```diff
@@ -1,4 +1,4 @@
-/*
+/*        auto timestamp = ROS2Interface::Get()->GetROSTimestamp();
```

Comment:
update: Looks like a typo that can be removed 😅

Reply:
Ups, sorry. Removed.